nototools/noto_lint.py

```diff
@@ -1051,6 +1051,38 @@ def check_ul_unicode_range():
 
             check_ul_unicode_range()
 
+            if os2_table.panose.bFamilyType != 2:
+              warn("head/os2/panose/family", "OS/2",
+                   "Panose family value is %s but expected 2" %
+                   os2_table.panose.bSerifStyle)
+
+            expect_serif = noto_font.style == 'Serif' or noto_font.family in [
+                'Cousine', 'Tinos']
+            expected_serif_range_str = '2-10' if expect_serif else '11-15'
+            serif_val = os2_table.panose.bSerifStyle
+            is_serif = 1 < serif_val < 11
+            if serif_val == 1:
+              warn("head/os2/panose/serif", "OS/2",
+                   "Panose serif value is no_fit (1) but expected 0, %s" %
+                   expected_serif_range_str)
+            elif serif_val != 0 and expect_serif != is_serif:
+              warn("head/os2/panose/serif", "OS/2",
+                   "Panose serif value is %s but expected %s" %
+                   (serif_val, expected_serif_range_str))
+
+            expect_mono = noto_font.is_mono or noto_font.family == 'Cousine'
+            expect_mono_range_str = '9' if expect_mono else '2-4'
```

Comment:
> It seems pretty likely to me that a test for bProportion == 9 is actually
> used somewhere to identify a monospace font.

In addition to Windows, Skia does that (I sent you a link to the code where it does in an email thread on the topic). 

> It's not clear to me that anyone uses the other proportion values to identify a condensed/wide font. 
> Do you know of some code that does?

I don't know any (other than monospace), but I thought it's just a bit tidier to mark 'condensed' with 6 or 8. Unlike other attributes, 'condensed' is a rather clear distinction, isn't it?